integration-tests/tests/src/tests/sync/mixed.ts

```diff
@@ -138,7 +161,7 @@ function describeRoundtrip({
       expect(typeof obj).equals("object");
       // Test the single value
       performTest(obj.value, this.value);
-      // Test the list of values
+      // Test the list of values  //TODO Maybe we don't need this?
```

Comment:
Isn't it worth testing that what we put in is also what we get out?

Reply:
I misunderstood a little bit the test and why the list was used. Now I understand what it was done (testing if a mixed value inside a list gets correctly around) and I'll add a similar one for a dictionary)